<ui version="4.0" >
 <class>Fetion_Login</class>
 <widget class="QWidget" name="Fetion_Login" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>281</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>311</width>
    <height>220</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LibFetion_image" >
   <property name="geometry" >
    <rect>
     <x>94</x>
     <y>498</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="login_image" >
   <property name="geometry" >
    <rect>
     <x>6</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ED_Fetion_Pwd" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>132</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string>QLineEdit {
      border-width: 1px;
      border-style: solid;
      border-radius: 4px;
  } </string>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="ED_Fetion_ID" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>89</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string>QLineEdit {
      border-width: 1px;
      border-style: solid;
      border-radius: 4px;
  } </string>
   </property>
   <property name="text" >
    <string comment="mobile number or fetion uid" />
   </property>
   <property name="maxLength" >
    <number>11</number>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Net_Setting" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>178</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="font" >
    <font/>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text" >
    <string>&lt;a href="http://www.libfetion.cn">Network Setting&lt;/a></string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="About_linux" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>198</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font" >
    <font/>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text" >
    <string>&lt;a href="http://www.libfetion.cn">about libfetion&lt;/a></string>
   </property>
   <property name="textFormat" >
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="openExternalLinks" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="remPWD" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>133</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text" >
    <string>remember password</string>
   </property>
  </widget>
  <widget class="QLabelMouse" name="BT_Login_Ok" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>183</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="text" >
    <string>login</string>
   </property>
  </widget>
  <widget class="QComboBox" name="loginState" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>89</y>
     <width>69</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string/>
   </property>
   <item>
    <property name="text" >
     <string>online</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>offline</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>away</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>busy</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>73</y>
     <width>251</width>
     <height>13</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>mobile number or fetion uid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>113</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text" >
    <string>please input password</string>
   </property>
  </widget>
  <widget class="QLabel" name="Login_State" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>160</y>
     <width>173</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>141</width>
     <height>21</height>
    </size>
   </property>
   <property name="toolTip" >
    <string/>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelMouse</class>
   <extends>QLabel</extends>
   <header>fxLableMouse.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ED_Fetion_ID</tabstop>
  <tabstop>ED_Fetion_Pwd</tabstop>
  <tabstop>loginState</tabstop>
  <tabstop>remPWD</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
